automation/src/test/resources/reference.conf

```diff
@@ -5,4 +5,4 @@ leonardo.gatkImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:0.0.4
 leonardo.baseImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-base:0.0.5"
 leonardo.oldWelderImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:6a783a5"
 leonardo.currentWelderImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:60e28bc" # This needs to be exactly the same in http/src/main/resources/reference.conf
-leonardo.bioconductor = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:0.0.4"
+leonardo.bioconductorImageUrl = "us.gcr.io/bioconductor-anvil/terra-jupyter-bioconductor@sha256:6bd0e493d38a5061f7621c0eb0fe383c3df1beb3432f93a4b570b9bcc59f129f"
```

Comment:
Just curious why we're using a sha instead of a tag..

Reply:
I think this is just for Nitesh's testing purpose. It needs to be updated with https://github.com/DataBiosphere/leonardo/pull/1155